pkg/archive/archive_unix.go

```diff
@@ -26,6 +26,9 @@ func fixVolumePathPrefix(srcPath string) string {
 // can't use filepath.Join(srcPath,include) because this will clean away
 // a trailing "." or "/" which may be important.
 func getWalkRoot(srcPath string, include string) string {
+	if srcPath == string(filepath.Separator) {
```

Comment:
FWIW this is not needed for the main fix. It is just fixing a weird input (`getWalkRoot("/", ".")` used to return `//.`, now just `/.`)

Reply:
Hmm, why can't we just use `path.Join(srcPath, include)` which should handle this and other similar cases? Say, https://play.golang.org/p/jG-JpZ-Kun5

Reply:
Nevermind, I missed the comment explaining why :man_facepalming: